docs/tech/rfc/0021-240711-evaluating-reliability.md

```diff
@@ -0,0 +1,161 @@
+# 0 - Summary 
```

Comment:
Could you add one `<h1>` at the top with the name of the RFC and then make the rest of these `<h2>`?

Reply:
Also at the top, add author name.